src/Entity/User.php

```diff
@@ -93,7 +93,7 @@ class User implements UserInterface
     public function __construct(UuidInterface $id, string $email, string $emailConfirmToken, array $roles)
     {
         $this->id = $id;
-        $this->email = $email;
+        $this->email = \mb_strtolower($email);
```

Comment:
this should also be added to unique email validator `Buddy\Repman\Validator\UniqueEmailValidator` to handle this case in UI layer

Reply:
OK!

Reply:
@akondas done